grayskull/pypi/pypi.py

```diff
@@ -59,21 +59,38 @@ def __init__(
         self["build"]["script"] = "<{ PYTHON }} -m pip install . -vv"
 
     @staticmethod
-    def _download_sdist_pkg(sdist_url: str, dest: str):
+    def _pkg_name_from_sdist_url(sdist_url: str):
+        if sdist_url.startswith(("http://", "https://")):
```

Comment:
Replace this check here (and elsewhere) with a call to a little method that, for now, only does this single check. For example:

```python
def _from_github(self, sdist_url):
    return sdist_url.startswith(("http://", "https://"))
```

```suggestion
        if self._from_github(sdist_url):
```

Reply:
Actually, do it like [this](https://github.com/ForgottenProgramme/grayskull/pull/6/files#r708028003)